src/App.vue

```diff
@@ -1,28 +1,36 @@
 <template>
-  <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+  <div>
+    <component :is="'Step' + stepNumber" :stepNumber="stepNumber" @parentFunc="updateForm" @backFunc="downStep"></component>
   </div>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld.vue'
+import Step1 from "./components/Step1";
+import Step2 from "./components/Step2";
+import Step3 from "./components/Step3";
 
 export default {
   name: 'App',
   components: {
-    HelloWorld
+    Step1,
+    Step2,
+    Step3,
+  },
+  data() {
+    return {
+      form: [],
+      stepNumber: 1
+    }
+  },
+  methods: {
+    updateForm: function(e) {
+      console.log(e);
+      this.form.push(e);
+      this.stepNumber++;
+    },
+    downStep: function(e) {
+      this.stepNumber = e - 1
+    }
```

Comment:
プロジェクト内でのコード規約は統一したいです。他の場所において 省略記法を使っていましたので、そちらに統一しましょう。
```suggestion
    updateForm(e) {
      console.log(e);
      this.form.push(e);
      this.stepNumber++;
    },
    downStep(e) {
      this.stepNumber = e - 1
    }
```